src/lib/__snapshots__/create-config.test.ts.snap

```diff
@@ -104,7 +104,7 @@ exports[`should create default config 1`] = `
       "projectStatusApi": false,
       "proxyReturnAllToggles": false,
       "responseTimeWithAppName": false,
-      "serviceAccounts": false,
+      "serviceAccounts": true,
```

Comment:
Could be a mistake from having the ENV variable set locally. Try recreating the snap with it off?

Reply:
Not exactly sure why this happens but it's not because of a local variable... :thinking:

Reply:
I just need to remember pushing my local branch :D